src/templates/en/2019/chapter_author.html

```diff
@@ -0,0 +1,24 @@
+{% for author in authors %}
```

Comment:
Rather than individual template files for each of these components, we can generate them in the existing chapter template file.

Similar to the render_byline macro in base_chapter.html.